### DISCUSSION TITLE
New User with User and Panel Extension

### DISCUSSION THREAD

**Discussion by nurbaitifitriyani** [mecha-cms/mecha#202] [Q&A] [answered]
Halo, Mas Taufik.

Saya mengunduh Mecha versi 2.6.4 dan memasang ekstensi 'user' dan 'panel' versi _stable_. Setelah membuat _username_ dan _password_, saya tidak dibawa ke halaman yang biasanya ada tautan profil/keluar. Mecha masih tetap di halaman untuk membuat user baru. Saya tidak bisa masuk/keluar.

Sementara itu, saya lihat di direktori 'lot', tidak muncul direktori 'user' beserta data-data (_username_ dan _password_) yang sebelumnya saya buat di laman user baru.

Saya mengira Mecha melalui peramban gagal mengepos username dan password baru ke dalam server Apache saya. Namun, saya coba menghapus user di Mecha versi lama yang saya gunakan, dan mencoba membuat username dan password baru, ternyata bisa mengepos data yang saya buat.

Server lokal saya adalah Apache versi 2.4 disertai PHP 7.4 di Debian GNU/Linux 11 Bullseye. Saya menaruh Mecha di direktori /home/user/public_html, bukan di root /var/www.

Apakah ini bug di sisi Mecha atau server saya tidak mengijinkan untuk mengepos melalui peramban?

Adakah solusi, jika memang server saya yang tidak mengijinkan untuk mengepos, bagaimanakah cara eksekusi di terminalnya.

Terima kasih atas diskusinya.

**Comment by taufik-nurrohman** [comment, marked as answer]
Mecha tidak sekompleks itu mba. Mungkin ada masalah permisi di akses folder. Coba set `DEBUG` mode ke `true` lalu coba buat _user_ lagi. Biasanya nanti akan muncul berkas `.\engine\log\error` kalau memang ada kegagalan.

**Reply by nurbaitifitriyani** [reply]
Benar, Mas. Ternyata soal akses permisi fail dan folder. Saya tes di web hosting secara daring, bisa buat user baru. Saya langsung cek akses folder lokal, ternyata `www-data` butuh dijadikan sebagai grup agar dapat mengakses fail dan folder.

**Reply by taufik-nurrohman** [reply]
Oke, udah _fix_ berarti ya!
